osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -379,8 +467,16 @@ public void DeleteItems(ISkinnableDrawable[] items)
 
         public Task Import(params string[] paths)
         {
+            if (menuBackgroundChooser.PopoverVisible)
+                return Task.CompletedTask;
```

Comment:
This early-return needs an explanatory comment as to why it exists at the least. But I find it really strange that this PR adds *two* import handlers, one of which just plays some sounds and closes the import popover, and the other does the actual import logic outside of the component responsible for actually selecting the background file.